Flutter/cartesian-charts/axis-types.md

```diff
@@ -65,17 +78,40 @@ By using the [`isInversed`](https://pub.dev/documentation/syncfusion_flutter_cha
 
     @override
     Widget build(BuildContext context) {
+        final List<ChartData> chartData = <ChartData>[
+            ChartData(2010, 10.53),
+            ChartData(2011, 9.5),
+            ChartData(2012, 10),
+            ChartData(2013, 9.4),
+            ChartData(2014, 5.8),
+            ChartData(2015, 4.9),
+            ChartData(2016, 4.5),
+            ChartData(2017, 3.6),
+            ChartData(2018, 3.43),
```

Comment:
Is the data is fine, check with the below image
![image](https://user-images.githubusercontent.com/19346074/154728657-4a19513a-d086-42fc-b71d-5783ee56b010.png)

Reply:
have added the proper image